MoneySplitter.Models/UserRole.cs

```diff
@@ -5,6 +5,7 @@ public enum UserRole
         IN_BEGIN,
         IN_PROGRESS,
         FINISHED,
+        USER_TRANSACTION,
```

Comment:
Not informational, maybe better rename to just MY or OWNER

Reply:
rename